extensions/amp-ad/0.1/test/test-concurrent-load.js

```diff
@@ -75,13 +75,27 @@ describes.realWin('concurrent-load', {}, env => {
       installTimerService(win);
     });
 
-    it('should throttle ad loading one per second', () => {
+    it('should throttle ad loading one per second', function* () {
       expect(is3pThrottled(win)).to.be.false;
       incrementLoadingAds(win);
       expect(is3pThrottled(win)).to.be.true;
       clock.tick(999);
+      yield;
```

Comment:
depending on the use cases. here we do have a handle of the promise, but in many real scenario, we don't.

also, `await` is not ES6 feature.

`yield` is actually nature to me. it means to `yield` to other code (other tasks in the event loop) to run first.